README.md

```diff
@@ -1,2 +1,172 @@
-# Hello
-Microverse Day-2 : Linters integrate with Git hub
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello Microverse project README</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> Hello Microverse is a simple project using html and css.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML5
+2. CSS
+3. HTML-CSS linters
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+1. Added index.html file
+2. Added stle.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Comming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+1. Code editor
+2. Web browser
+3. Git
+
+### Setup
+
+Clone this repository to your desired folder:
+Run the below command:
+
+```sh
+cd <your disered folder name >
+git clone git@github.com:HossainAraf/Hello.git
+```
+### Install
+
+Install with below command:
+
+```sh
+  cd <your disered folder name >
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following:
+
+Open index.html file either directly or using live server 
+
+### Run tests
+
+Comming soon
+
+### Deployment
+
+Comming soon
+
```

Comment:
[OPTIONAL]
- [ ] As suggested in a previous comment, kindly delete any section that cannot be customized with your project until you need it.

---

- _NB: Once you delete a section from the `README`, make sure you delete it also from the table of contents._